### ISSUE TITLE
Can not see crash logs from other apps?

### ISSUE THREAD

**Issue by justin3527** [DarielChen/DCLog#1]
hi, im student 
please give me advice

Now, i hope to make the app  that monitor the crash log of all apps in my device 
i can find the DCLog while i search the library
but i think that this app check only inside of app

Can not see crash logs from other apps? or did you know solution of this problem?

please help me

**Comment by DarielChen** [comment]
NO, every app in iOS device is a sandbox, you can not get anything out from it.

**Comment by justin3527** [comment]
thank you very much ,sir

**Comment by justin3527** [comment]
I have one more question.
can i get device log like to device tab of xcode
